### DISCUSSION TITLE
Request for 2/3 center

### DISCUSSION THREAD

**Discussion by floriandulzky** [rxhanson/Rectangle#1340] [Ideas]
Idk if it was already discussed (couldn't find it).

I am working on a wide-screen and I very often use the feature "1/2 center" and then increase the window (with the "larger" feature) to roundabout 2/3 of the screen size. 

It would be very nice to do this within a single shortcut.

**Comment by rxhanson** [comment]
Here's an oldy where this is talked about: https://github.com/rxhanson/Rectangle/issues/52
